docs/guides/basic-rbac.mdx

```diff
@@ -3,267 +3,303 @@ title: Implement basic Role Based Access Control (RBAC) with metadata
 description: Learn how to leverage Clerk's publicMetadata to implement your own basic Role Based Access Controls.
 ---
 
-To control which users can access certain parts of your application, you can leverage Clerk's [roles](/docs/organizations/roles-permissions#roles) feature. Although Clerk offers a roles feature as part of the feature set for [organizations](/docs/organizations/overview), not every app implements organizations. This guide will cover a workaround for setting up a basic Role Based Access Control (RBAC) system for products that don't use Clerk's organizations or roles.
+To control which users can access certain parts of your application, you can leverage Clerk's [roles](/docs/organizations/roles-permissions#roles) feature. Although Clerk offers a roles feature as part of the feature set for [organizations](/docs/organizations/overview), not every app implements organizations. **This guide covers a workaround to set up a basic Role Based Access Control (RBAC) system for products that don't use Clerk's organizations or roles.**
 
-This guide assumes that you are using Next.js App Router. The concepts here can be adapted to Next.js Pages Router and Remix.
+This guide assumes that you're using Next.js App Router, but the concepts can be adapted to any React-based framework.
```

Comment:
This is not really true -- this is leveraging server side checks on the role and then redirecting the user from the route. A React app can't do this. To do this on React you would need to leverage an Express or other server side app to handle much of the logic and the implementation would be different. I would revert to the previous statement.

Reply:
[c2ee7a1](https://github.com/clerk/clerk-docs/pull/1560/commits/c2ee7a1f4a008ea6e51f4d097dda612f213575cb)